LEANNE_WAKE_STEERING2016.md

```diff
@@ -0,0 +1,50 @@
+ 
+You can find me in the Department of Geography in the [University of Northumbria at Newcastle, UK](https://www.northumbria.ac.uk/about-us/our-staff/w/dr-leanne-mary-wake/).  In 2010 I obtained my PhD in Geophysics from Durham University, UK.  I would describe myself, broadly, as an Earth system modeller. I try to understand the workings of a complex world through the 0s and 1s of code but I have been known get my hands dirty with fieldwork!  
+
+## Relevant experience 
+
+Having been exposed to a wide variety of code instruction techniques (from “you’re on your own, mate” to “Paired Programming”), I became motivated to find out what the right method was to introduce someone to code. I became involved in Software Carpentry (SWC) via a [Fellowship in 2014] (http://software.ac.uk/fellows/leanne-mary-wake) with the Software Sustainability Institute. My main interest is software education - see [here](http://www.software.ac.uk/introduction-fortran-95). I qualified as a [SWC instructor](http://www.software.ac.uk/swc-instructor-training) in 2015 and subsequently co-led a workshop at [St Andrew’s University] (http://lmwake.github.io/2015-06-18-StAndrews/) in the summer of 2015. 
+
+## How I will contribute to the growth of Software Carpentry
+
+* *Global Expansion*:
+
+Involvement with the Software Sustainability Institute and SWC has convinced me that SWC should be made available to as wide a user-base as possible.  As a geographer, I was drawn to this section of the SWC website, and saw that SWC’s [global coverage](http://software-carpentry.org/workshops/past/) could be improved. Imagine the scenario: 
+
+**Institute to Funder:** ‘I’d like some money to host a Software Carpentry workshop, please’
+**Funder:** ‘What percentage increase in coding ability can we expect from this investment?’
+**Institute:** ‘Ummmmm….’
+
+Metrics and Quality Assurance (QA) are a part of everyday life but also huge bugbears for most in academia. However, I believe they can be used as agents for change for SWC.
+
+**As part of my tenure on SWC’s steering committee, I’d like to work with interested parties both inside and outside SWC towards development of a ‘influence metric’ that SWC can use to show the positive impact of it’s teaching.  Let’s turn those post-it notes into points!** 
+
+* *Course Content:*
+
+**Parallelisation:** Making your code more efficient. You have built a car - how do you turn it into a Ferrari?  How many folks do you know have a fancy multi-core machine, yet only use a single core? SWC is contributing towards reproducible science, how about efficient science?
+
+* *Organisational Duties/Miscellaneous:*
+
+I am willing to accept any admin duties to ensure fluidity and expansion of the organisation, e.g.:
+
+-**Maintaining a live instructor database** indicating temporal availability of instructors. 
+
+-**Recruitment:** Workshop and participants number are increasing; Instructor numbers are [tailing off.] (http://software-carpentry.org/about/)
```

Comment:
A stable instructor count should be able to maintain a linear learner increase ;).  But yeah, the more, the merrier :).